plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -290,6 +324,8 @@ func init() {
 		m.Path = path
 	}
 	m.Timeout = internal.Duration{Duration: time.Second * 20}
+	m.UseCache = true
```

Comment:
* caching uses less then 10kB per host for me.
* just in case cache is a issue for whatever unseen reason for someone i like it to be optional
* I don't know the inner workings of ipmitool so idk

Reply:
@reimda Are you ok with this explanation?

Reply:
The ipmitool program itself defaults to no caching and requires their users to turn it on with a command line switch.  I'm not comfortable with having telegraf change this and cache by default.  I understand that caching has worked well for @DEvil0000 but I don't feel like we have an explanation of whether switching caching on has the potential to cause trouble for existing telegraf users who upgrade.

I'm fine merging this PR if we make caching opt-in by defaulting to "use_cache = false" (and fix the typos you brought up in your review)

Reply:
okay, will change it to be turned off by default

Reply:
Thanks!